.travis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     before_script:
     - |
       set -ex
-      git clone --depth 1 https://github.com/puppetlabs/puppetlabs-motd puppetlabs-motd
+      git clone --depth 1 -b maint/DO-NOT-MERGE/pin_stdlib_v6_6_0 https://github.com/sanfrancrisko/puppetlabs-motd puppetlabs-motd
```

Comment:
Until the [dependencies](https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/b2d6be9e58fc44ba78f3d0cb9b556db203b3d35e/metadata.json#L125-L128) of `puppetlabs-puppet_agent` are bumped, this workaround is required - see [here](https://github.com/puppetlabs/puppetlabs-motd/compare/main...sanfrancrisko:maint/DO-NOT-MERGE/pin_stdlib_v6_6_0)